app/build.gradle

```diff
@@ -41,6 +39,9 @@ dependencies {
     debugCompile 'com.squareup.leakcanary:leakcanary-android:1.5.1'
     releaseCompile 'com.squareup.leakcanary:leakcanary-android-no-op:1.5.1'
     testCompile 'com.squareup.leakcanary:leakcanary-android-no-op:1.5.1'
+
+    compile "io.reactivex.rxjava2:rxkotlin:$rx_version"
+    compile "io.reactivex.rxjava2:rxandroid:$rx_android_version"
```

Comment:
It was an omission on my part in the transition from this PR being written in Kotlin -> Java.  We should follow the documentation you pointed out and track versions of RxJava and RxAndroid individually.  Keep an eye open and make sure I don't miss it in the _actual_ implementation!